libraries/mbed/targets/hal/TARGET_NORDIC/TARGET_MCU_NRF51822/TARGET_NRF51822_MKIT/PinNames.h

```diff
@@ -61,7 +61,7 @@ typedef enum {
     p28 = 28,
     p29 = 29,
     p30 = 30,
-//    p31=31,    
+    p31 = 31,    
```

Comment:
According to the [user guide](https://www.nordicsemi.com/eng/nordic/download_resource/30462/1/50872152), it is.